### PR TITLE
[docs] update documentation with the graph subgraph setup steps

### PR DESCRIPTION
**Pull request by benschac**
## What does this PR do and why?

I went through the documentation to get the project working locally on my machine. I assumed we would also want to document the subgraph setup too.

Adds setup up steps for Juicebox subgraph initial step up.

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](../CONTRIBUTING.md#approval-guidelines) for this PR.

 I tried looking at the approval guidelines, but a don't have access to the docs
 

 
- [x] I have tested this PR in [all supported browsers](../CONTRIBUTING.md#browser-support).

### CODE REVIEW

**Comment by johnnyd-eth** [comment]
Great addition!

**Comment by benschac** [comment]
updating the documentation to direct contributors to the Disord for subgraph URLs. https://discord.com/channels/939317843059679252/939705688563810304/954851207032209468

File: README.md [hunk 1]
```diff
@@ -98,6 +98,18 @@ Take the following steps to set up Blocknative for local development:
...
    `.env` file.
 
+#### The Graph
+
+Juicebox uses [The Graph](https://thegraph.com) to query the Ethereum network
+using a GraphQL API.
+
+Take the following steps to set up Juicebox's subgraph for local development:
+
+1. Go to
```
Comment (aeolianeth) [inline README.md:108]: **praise** super clear, thank you!

**Review by aeolianeth — APPROVED** [review]
Thank you @benschac! Really appreciate additions to our onboarding. Approving and merging.

File: README.md (hunk 1 above)
Comment (benschac) [reply README.md:108]: ty

**Pull request merged**